python/extension/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ endif()
 include(HandleLLVMOptions)
 include(AddMLIRPython)
 
+if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND NOT APPLE)
+  add_link_options(-Wl,--no-as-needed)
```

Comment:
Why is this needed?

Reply:
This can be dropped, left over from debugging